workshop.pl

```diff
@@ -357,23 +368,23 @@ sub arg_handler {
     } elsif ($opt_name eq "config") {
         $args{'config'} = $opt_value;
 
-        if (! -e $args{'config'}) {
-            die("--config must be a valid file [not '$args{'config'}']");
-        }
+        #if (! -e $args{'config'}) {
+            #die("--config must be a valid file [not '$args{'config'}']");
+        #}
```

Comment:
just delete it

Reply:
I think a lot of these should not have shown up with all commits.  Maybe all commits are not here, I'll check